ActiveDocuments/execCommand.html

```diff
@@ -710,17 +672,12 @@ <h2 id="methods-to-query-and-execute-commands">
                 would not be <a href="#enabled">enabled</a>.
               </p>
             </li>
-            <li>[=Dispatch=] an [=event=] at <var title="">affected editing
-            host</var> that uses the <a>EditingBeforeInputEvent</a> interface.
-            The event's {{Event/type}} attribute must be initialized to
-            "beforeinput"; its {{Event/isTrusted}}, {{Event/bubbles}}, and
-            {{Event/cancelable}} attributes must be initialized to true; its
-            <code title="dom-EditingBeforeInputEvent-command"><a href=
-            "#dom-editingbeforeinputevent-command">command</a></code> attribute
-            must be initialized to <var title="">command</var>; and its
-            <code title="dom-EditingBeforeInputEvent-value"><a href=
-            "#dom-editingbeforeinputevent-value">value</a></code> attribute
-            must be initialized to <var title="">value</var>.
+            <li>[=Dispatch=] an [=event=] at <var>affected editing host</var>
```

Comment:
Nit: I wonder if we should use the "fire an event" algo here instead? Also, does all this happen sync or at these queued tasks (and if so, what task source)?

Reply:
It happens synchronously on both Firefox and Chrome. Check out https://codepen.io/SaschaNaz/pen/MWaXLQE to see the behavior.